lib/rules/cli/4.4.yaml

```diff
@@ -208,12 +211,15 @@
     :output: --output <value>
     :template: --template=<value>
 :create_job:
-  :cmd: oc create job <name>
+  :cmd: oc create job <name> <global_options> <options>
```

Comment:
No need the change in this line now.
Also please sort the options in alphanumeric.

Reply:
No , must order the global_options and options . because if without the order '--kubeconfig' will not take effect .